### ISSUE TITLE
2.5.2 breaks Guild Wars 1

### ISSUE THREAD

**Issue by KayTaneJupiter** [doitsujin/dxvk#4546]
### Software information
Guild Wars 1

### System information
- GPU: AMD
- API: d3d9
- Wine version:  10 rc3
- DXVK version: 2.5.2

Causes unreadable artifact garbage screen

2.5.1 works fine

Good vs Bad examples attached

![Screenshot_20241223_212212](https://github.com/user-attachments/assets/b4937133-c9a6-4cea-a4e6-23d9c1ff3e8a)

![Screenshot_20241223_211329](https://github.com/user-attachments/assets/c7bbfa5f-707a-4c80-8c99-7e98eda18797)

**Comment by K0bin** [comment]
This is most likely already fixed in master. 2.5.2 unfortunately shipped with a bug that broke a lot of old games.

**Comment by casasfernando** [comment]
> This is most likely already fixed in master. 2.5.2 unfortunately shipped with a bug that broke a lot of old games.
> 
> https://github.com/doitsujin/dxvk/actions/runs/12435734651

I was about to open a new issue for Prince of Persia - The Sands of Time that is also broken after updating to DXVK 2.5.2.

@K0bin, any change to push a new release (2.5.2-1 or 2.5.3) including the fix ASAP?
Launchers like Heroic will automatically update DXVK as soon as a new version is available (as far as I know without notifying the user) and potentially breaking the games (this is what happened in my case) so having a new release with the fix can prevent many user from hitting this issue out of the blue and opening new issues for this problem.

Thanks

**Comment by K0bin** [comment]
It's probably unlikely that there will be a new release over the holidays.

**Comment by casasfernando** [comment]
Indeed. The issue is fixed for Prince of Persia - The Sands of Time with https://github.com/doitsujin/dxvk/actions/runs/12435734651

Note for Heroic Games Launcher users (Linux) out there. Before starting any new games you may want to run the steps below. For games that are already broken, after running the steps below, you can simply uncheck the DXVK install/autoupdate wait for 30 seconds or so and check it again (this will remove and reinstall DXVK for the game with the fixed build):

1. Download the build including the fix: https://github.com/doitsujin/dxvk/actions/runs/12435734651
2. Extract the `x32` and `x64` directories from the ZIP file
3. Replace the `x32` and `x64` directories in the following paths (depending on the HGL release you are using) with the ones extracted from the ZIP file:
- HGL *Flatpak*: `~/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/dxvk/dxvk-2.5.2`
- HGL *non-Flatpak*: `~/.config/heroic/tools/dxvk/dxvk-2.5.2`

Thanks
